tutorial-server/notebooks/CactusTutorial.ipynb

```diff
@@ -101,7 +101,32 @@
     "os.environ[\"OMPI_MCA_rmaps_base_oversubscribe\"] = \"true\"\n",
     "os.environ[\"OMPI_MCA_hwloc_base_binding_policy\"] = \"none\"\n",
     "os.environ[\"OMPI_MCA_btl_vader_single_copy_mechanism\"] = \"none\"\n",
-    "import scrolldown"
+    "#import scrolldown\n",
```

Comment:
I would like to keep scrolldown. It could be installed with pip, or a try/except could be added.

Reply:
I will add a try/except.